index.html

```diff
@@ -82,21 +91,34 @@
         This feature is currently testing on Greek and Catalan Wikipedia. It required 3 different layouts to support a cohesive image text layout. The key question was ‘How much content is needed for a user to get context?'
       </p>
     </section>
+    <section class="tags para">
+      Team - engineering, community liason, design
+    </section>
   </article>
   <article id="android" class="page focus">
-    <a href="#" class="backout">&larr;</a>
+    <a href="#" class="backout">
+      <img src="images/backbutton.png" />
+    </a>
     <img src="images/tile3-details.png" />
     <section class="headline">
       Android App
     </section>
     <section class="details para">
       <p>
-        The native Android app is a valuable experimentation space for us due to the large alpha and beta channels, where we can run a/b tests for content experiments. So far we have reliably tested sharing of facts and quick link previews. The platform integration with Google Now is useful for us to show wikipedia content in search or in the context of other properties.
+        The native Android app is a critical experimentation space for us due to the large alpha and beta channels, which are valuable feedback channels. The platform integration with Google Now is useful for us to show wikipedia content in search or in the context of other properties.
       </p>
     </section>
+    <section class="tags para">
+      Team - engineering, design, legal |
+      Press -
+      <a>New York Times</a>,
```

Comment:
these don't have `href` attributes yet, but I assume you'll add them in future work.